policies/project-director-election-process.md

```diff
@@ -0,0 +1,68 @@
+# Project Director Election Process
+
+## Goals
+
+The purpose of this policy is to outline a clear and efficient process for electing new Project Directors to represent the Rust project on the board of the Rust Foundation. The process aims to achieve the following goals:
+
+* Select candidates who fulfill the outlined candidate criteria in the [role description document](TODO: link), focusing on shared power and new leadership rather than solely prioritizing the best candidate.
+    * Strive for tolerance and inclusivity, aiming for the overlap of tolerance rather than the overlap of preference. In other words, we want to prioritize a selection that everyone can accept over selecting anyone's first choice.
+* Conduct the selection process in a time-efficient manner.
+* Incorporate input from a wide range of stakeholders across the entire Rust project.
+* Ensure a diverse pool of qualified candidates for selection.
+* Create a psychologically safe environment that values minority perspectives and encourages positive feedback, encouragement, and tolerance rather than preference or criticism.
+* Ensure participants feel positive about their interactions and the execution of the process.
+* Ensure the process is compliant with Rust Foundation bylaws and by extension Delaware corporation law.
+
+## Description
+
+### Setup
+
+* The council selects a facilitator by consent:
+    * Any team member can suggest a facilitator or volunteer themselves.
+    * The facilitator must not be a nominee in the election and must be someone everyone trusts to set aside personal preferences during the election process.
+    * **Goals of the facilitator role**
+        * Clearly documenting and communicating the process and announcing and supporting its execution.
+        * Tracking, reviewing, and documenting the effectiveness of the process.
+* Review Role Description ([example](https://github.com/rust-lang/rfcs/pull/3392#issuecomment-1505697944)). The role description must contain the following content at a minimum:
```

Comment:
I'm confused. I believe this used to be "Create and review" but it was changed to just "review" because the Foundation staff had already created a role description. However, that role description doesn't seem to be linked anywhere. Can we link to it? 

If we've decided that we actually don't want to automatically use that role description, than can the start of this line be reverted to "Create and review"?

Reply:
I think we'll need to modify the role description to include some additional detail, but otherwise I think we should leave it as simply "review" on the expectation that we will have a role description alongside this document. I'm going to take the existing role description and convert it to a markdown document, pull it into this PR, then start expanding on it.

Reply:
Noting that it isn't, at present, a _Project Director_ role description, it's a _Rust Foundation Board Director_ role description - which is to say, it applies to Member Directors as well. By all means create an addendum for expectations of Project Directors specifically, but be wary of making changes to the main role description unless those changes would also apply smoothly to a Member Director carrying out their role on the board.

Reply:
aah that's a good point. In that case I'll setup a Project Director specific role description and plan on linking to the general board director role and responsibilities document